lib/kms.js

```diff
@@ -6,6 +6,31 @@ import * as brHttpsAgent from 'bedrock-https-agent';
 import {getAppIdentity} from 'bedrock-app-identity';
 import {KeystoreAgent, KmsClient} from '@digitalbazaar/webkms-client';
 
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock.init', async () => {
+  if(!(bedrock.config && bedrock.config.tokenizer &&
+    bedrock.config.tokenizer.kms && bedrock.config.tokenizer.kms.baseUrl)) {
+    throw new TypeError(
+      'bedrock.config.tokenizer.kms.baseUrl config value is required.');
+  }
+});
+
+/**
+ * Creates a new WebKMS keystore for tokenizer use.
+ * Requires bedrock.config.tokenizer.kms.baseUrl to be set.
```

Comment:
I don't think we need this comment on the `...baseUrl` thing now.

Reply:
Removed.